ads/aja.js

```diff
@@ -14,28 +14,18 @@
  * limitations under the License.
  */
 
-import {loadScript, validateData} from '../3p/3p';
+import {writeScript} from '../3p/3p';
 
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function aja(global, data) {
 
-  // ensure we have valid widgetIds value
-  validateData(data, ['widgetids']);
```

Comment:
If `data-ssp-code` is a required param, you may still want to validate data.

Reply:
validate data...

Is there a check as to whether there is?

sorry..

Reply:
Using the `#validateData()` helper method. 
`validateData(data, ['sspCode'])`, depending on if the `data-ssp-code` is required or not

Reply:
If not, is it a function for announcing?

Reply:
How do you want to handle if there's no `ssp-code`? What `validateData` does is simply checking the existence of `ssp-code` and throw a user error if it's not there. I guess if you want some kind of fallback, you either check with `if/else` or use `try/catch`

Reply:
understood!
I will add it on weekdays!

Reply:
An error occurred😭

Reply:
![image](https://user-images.githubusercontent.com/46767673/53707158-1ff6b800-3e71-11e9-87be-dc018503c24f.png)

...?

Reply:
maybe fix it